### ISSUE TITLE
"Using Bridge" instructions don't describe all cases

### ISSUE THREAD

**Issue by Fang-** [urbit/docs#726]
From `/using/operations/using-bridge`:

> On the Admin page, click the Download Arvo Keyfile button.

As observed in urbit/bridge#343, it's not guaranteed that the user can do this right away. Edge cases include: not having keys configured yet, or not logging in with an Urbit Wallet. The description given works fine for people who came on through the invite flow, but is a bit lacking for those edge cases. Maybe say a few more words here, or link to other docs if we have them.

**Comment by brianmcmichael** [comment]
Purchased a couple of urbit ID's a while back on OpenSea and logged in to a local instance of the bridge using metamask.  From there it's not obvious what to do and the "download keys" option is greyed out. Some googling led me to #343, which is closed, but leads to here.

I did download a zip file with some images in it but it's not clear where to go next. Bridge docs seem light on this step.

**Comment by Fang-** [comment]
For @brianmcmichael and whoever else might be confused here, the missing step here is roughly:

Log in with an account that can manage the point, then go to admin -> edit permissions -> networking keys, wait for the transaction to finish, download the keyfile, then boot your urbit using it.

**Comment by danielalexiuc** [comment]
To clarify, you need to go to:

admin -> edit permissions -> networking keys -> Reset Networking Keys

Then wait for the transaction to finish.

**Comment by RokoMijic** [comment]
Yeah, the docs are pretty terrible, I have to say. After fiddling around for a while I managed to get things working, but if you come from Metamask the flow doesn't work. There are also some confusing naming inconsistencies. Needs to be rewritten with help from someone competent but new to Urbit.
